src/posts/dto/update-post.dto.ts

```diff
@@ -1,4 +1,4 @@
 import { PartialType } from '@nestjs/swagger';
-import { CreatePostDto } from './createPost.dto';
+import { CreatePostDto } from './create-post.dto';
```

Comment:
So we'll have all files with -, not camel case, right?

Reply:
To be honest, I don't have a strong opinion about casing. This is a default rule added by the eslint unicorn plugin(it has a nice set of good practice rules). If you want, we can disable this rule or add another one for the file naming convention.

Reply:
Nope, good to me!! Don't have a strong opinion on this either, so lets keep this default rule